auth/server/src/main/java/com/seagates3/authserver/AuthServerConfig.java

```diff
@@ -131,19 +133,43 @@ public static void logConfigProps() {
      * @throws GeneralSecurityException
      */
     public static void loadCredentials() throws GeneralSecurityException, Exception {
-
-        Properties authServerConfig = AuthServerConfig.authServerConfig;
-        String encryptedPasswd = authServerConfig.getProperty("ldapLoginPW");
-        Path keyStoreFilePath = getKeyStorePath();
-        PrivateKey privateKey = JKSUtil.getPrivateKeyFromJKS(
-                                keyStoreFilePath.toString(), getCertAlias(),
-                                getKeyStorePassword());
-        if (privateKey == null) {
-             throw new GeneralSecurityException("Failed to find Private Key ["
-                + keyStoreFilePath + "].");
-        }
-        ldapPasswd = RSAEncryptDecryptUtil.decrypt(encryptedPasswd,
-                     privateKey);
+       logger = LoggerFactory.getLogger(AuthServerConfig.class.getName());
+       String keystorePasswd = null;
+       String encryptedPasswd = authServerConfig.getProperty("ldapLoginPW");
+       Path keyStoreFilePath = getKeyStorePath();
+       try {
+         String cmd = authServerConfig.getProperty("cortexSecCmd");
+         logger.info("Command is : " + cmd);
```

Comment:
I think we should log this information at debug level instead of info !

Reply:
This is still in testing mode.. Not refined yet